service/controller/v15/key/key.go

```diff
@@ -108,6 +108,8 @@ func BucketName(customObject v1alpha1.AWSConfig, accountID string) string {
 	return fmt.Sprintf("%s-g8s-%s", accountID, ClusterID(customObject))
 }
 
+// TODO what should the path look like when we want to use the version bundle
+// version of the operator?
 func BucketObjectName(templateVersion string, prefix string) string {
 	return fmt.Sprintf("cloudconfig/%s/%s", templateVersion, prefix)
 }
```

Comment:
When I did not miss something critical this should be the place where we want to change the S3 object name. It should contain the version bundle version. How should the path look like though? I would provide a PR based on this one with further changes.

Reply:
Yes as far as I recall this is the right place. The current paths with k8scloudconfig look like 

```
/cloudconfig/v_3_2_5/master
/cloudconfig/v_3_2_5/worker
```
To me adding the version bundle version but keeping the k8scloudconfig version makes sense.

```
/cloudconfig/3.4.0/v_3_2_5/master
/cloudconfig/3.4.0/v_3_2_5/worker
```

The S3 URL for small cloudconfig will be unique but if we change the k8scc version in a WIP release the paths will still be correct.

Reply:
Thought on this bit more and I think `/cloudconfig/3.4.0/cloudconfig/v_3_2_5/worker` is better.

It's useful that we version the S3 URLs and in future we might need to store other types of assets. WDYT?

Reply:
Damn its too hot here! 😆 

Yes making the root `/version/` makes way more sense. What Pawel said. 😄

Reply:
Cool. Thanks gang. Will address this separately.